### ISSUE TITLE
A11y_code.visualstudio.com_Updates_March 2k19_ScreenReader: "Here" link name should be more descriptive.

### ISSUE THREAD

**Issue by Charanrajkb** [microsoft/vscode-docs#2793]
“[Check out Accessibility Insights!](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx
) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

Environment Details:
URL: [https://code.visualstudio.com](url)
Windows Version: Windows10
Browser Version:
Microsoft Edge 44.18362.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : NVDA2019.1.1
MAS Violated: MAS 1.3.1
MAS violated: MAS2.4.4

Repro Steps:
Navigate to the URL: [https://code.visualstudio.com/updates/v1_33](https://code.visualstudio.com/updates/v1_33)

follow below steps:
1. Launch the application Url : [https://code.visualstudio.com]([](https://code.visualstudio.com))
2. Navigate to "Updates" link.
3. From left navigation, select "March 2k19" link.
4.Start narrator.
5.Using Tab navigate to the here link present under the "Debug toolbar contribution support positioning" Section.

Actual:
Screen reader does not read more descriptive for the link "Here".
Narrator reads as link here.

Expected:
Screen reader must announce descriptive for the link Here.
Like when the narrator focus is on the link here must read click here to navigate to more information about sorting inside group.

Recommendations:
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

User Impact:
Screen reader user will get confused while navigating through link and does not come to know what's the link purpose and where it is navigating to.

MAS Reference:
MAS 1.3.1 -
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4})

MAS2.4.4:
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={06a891af-210c-4c6d-bb3e-92babab209e7}](url)

Attachment Reference:
[A11y_code.visualstudio.com_Updates_March 2k19_ScreenReader1.pptx](https://github.com/microsoft/vscode-docs/files/3319830/A11y_code.visualstudio.com_Updates_March.2k19_ScreenReader1.pptx)
